pvanalytics/tests/features/test_clearsky.py

```diff
@@ -1,10 +1,14 @@
 """Tests for feature labeling functions."""
 import pytest
 import pandas as pd
+import pvlib
 from pvanalytics.features import clearsky
 
+from pkg_resources import parse_version
+is_old_pvlib = parse_version(pvlib.__version__) < parse_version("0.9.0")
 
-@pytest.mark.skip(reason="GH #105")
+
+@pytest.mark.skipif(is_old_pvlib, reason="GH #105")
```

Comment:
Is this sufficient for #105?  It's not clear to me what the outcome of that issue was.

Reply:
I'm OK with marking this test to be skipped, as a way to get beyond it, but I think the #105 is still relevant. IIRC, the action here is to replace the expected result in the test. The test was created using pvlib v0.8.1, which had a bug corrected in v0.9.0 via [1242](https://github.com/pvlib/pvlib-python/pull/1242).

Reply:
What test values are there to update?  These tests don't seem to use particularly precise hardcoded values anywhere, unless I'm missing something.  

An aside: the commit message on 09e93a0 will automatically close #105, and I'm not sure how to prevent that now, so we'll have to manually reopen that issue if this is merged.